cernopendata/templates/cernopendata_pages/front/main.html

```diff
@@ -25,13 +25,14 @@ <h2>
           type="text"
           class="form-control search"
           name="q"
+          aria-label="search"
         />
         <div>
           <button type="submit">Search</button>
           <div class="cod-home-image">
             <img
               src="{{url_for('static', filename='assets/img/header-image.svg')}}"
-              alt=""
+              alt="cod-home"
```

Comment:
I would probably put some text with meaning or `alt=''`... I don't think `cod-home` is useful....

Reply:
```suggestion
              alt=""
```